src/databaseScripts/CreateTables.sql

```diff
@@ -0,0 +1,75 @@
+
+USE  AgileSprints_SamB;
+DROP TABLE Employees;
+DROP TABLE Job_Roles;
+
+
+CREATE TABLE if not exists Capabilities(
+	ID TINYINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+	CapabilityName varchar(15)
+);
+
+CREATE TABLE if not exists Bands(
+	ID TINYINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    BandLevel TINYINT,
+    BandName varchar(20)
+);
+
+CREATE TABLE if not exists JobRoles(
+	ID TINYINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    BandID TINYINT,
+    CapabilityID TINYINT,
+	KainosJobTitle varchar(30),
+	JobFamily varchar(50),
+	JobSpecification varchar(255),
+	JobSpecLink varchar(150),
+    FOREIGN KEY(BandID)
+		REFERENCES Bands(ID),
+	FOREIGN KEY(CapabilityID)
+		REFERENCES Capabilities(ID)
+);
+
+
+CREATE TABLE if not exists Trainings(
+	ID TINYINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    TrainingName varchar(80),
+    IsObligatory BIT -- Flag if the training is obligatory
```

Comment:
Can remove the comments - here and in other places

Reply:
done